src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,39 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
```

Comment:
2. **Documentation:** It would be beneficial to add some method-level documentation using comments to describe what each method does, especially for more complex logic in real-world scenarios.